src/components/editor/page-properties.tsx

```diff
@@ -469,6 +573,20 @@ const PageProperties = ({ pageNumber, surveyId }: Props) => {
         onChange={handleBackgroundChange}
         value={PageUtils.getPageBackground(elementsToEdit, pageToEdit.properties) ?? ""}
       >
+        <MenuItem>
+          <Button startIcon={<AddCircle />} onClick={handleAddNewImageClick}>
+            {strings.editSurveysScreen.editPagesPanel.addNewImage}
+          </Button>
+        </MenuItem>
+        {/* New images are included here along side the old ones for now */}
```

Comment:
Make this comment more descriptive. E.g. explain that the color "images" are listed as previously